README.md

```diff
@@ -66,14 +66,17 @@ Create a `env.json` file with the following content **(this file can also be obt
     ],
     "login_users": [
         {"username": "scraping-site-username", "password": "scraping-site-password"}
-    ]
+    ],
+    "mock_mode": true
```

Comment:
Should we keep by default on false it might cause confusion

Reply:
Yeah, having mock mode on by default would probably be better, this way people will have to explicitly turn it off and it keeps it from causing confusion.